crates/pathfinder/src/state/sync.rs

```diff
@@ -368,9 +373,11 @@ where
                     .map(|block| (block.number, block.hash, block.state_commmitment));
 
                     let (new_tx, new_rx) = mpsc::channel(1);
+
+                    let websocket_txs = WebsocketSenders::default();
```

Comment:
Default can have different capacity from the original one passed into the `sync()` function in the first place.

Reply:
`tokio::broadcast` has an `Arc` internally, so `clone()` is cheap

Reply:
<img align='left' width="50" src="https://github.com/eqlabs/pathfinder/assets/11048263/4438e0ab-5b7a-4f0e-a9c8-819bdc332942">
Don'no how this managed to get through, It shouldn't be...
This basically means it has no link with the original senders and nobody will ever listen/broadcast anything to it.

Reply:
It's fixed now.